services/cd-service/pkg/notify/notify.go

```diff
@@ -53,3 +56,31 @@ func (n *Notify) Notify() {
 		}
 	}
 }
+
+func (n *Notify) SubscribeChangesApps() (<-chan []string, Unsubscribe) {
```

Comment:
please create a type for either `chan []string` or `[]string` to make clear that this is a list of app names and nothing else.